app/assets/stylesheets/forever_style_guide/base/_typography.scss

```diff
@@ -13,53 +13,50 @@ h6,
 p {
   color: color('text');
 }
-
-h1 {
+h1, .heading-large {
   font-family: $font-face-gotham;
   font-weight: 500;
-  font-size: $font-size-700;
+  font-size: $font-size-500;
 
-  @media (max-width: $screen-sm-max) {
+  @media (min-width: $screen-sm) {
     font-size: $font-size-600;
   }
-  @media (max-width: $screen-xs-max) {
-    font-size: $font-size-500;
+
+  @media (min-width: $screen-md) {
+    font-size: $font-size-700;
   }
 }
-h2 {
+h2, .heading-medium {
   font-family: $font-face-gotham;
-  font-weight: 500;
-  font-size: $font-size-600;
+  font-size: $font-size-400;
 
-  @media (max-width: $screen-sm-max) {
+  @media (min-width: $screen-sm) {
     font-size: $font-size-500;
   }
-  @media (max-width: $screen-xs-max) {
-    font-size: $font-size-400;
+
+  @media (min-width: $screen-md) {
+    font-size: $font-size-600;
   }
 }
-h3 {
+h3, .heading-small {
   font-family: $font-face-gotham;
-  font-weight: 500;
-  font-size: $font-size-500;
+  font-size: $font-size-300;
 
-  @media (max-width: $screen-sm-max) {
+  @media (min-width: $screen-sm) {
     font-size: $font-size-400;
   }
-  @media (max-width: $screen-xs-max) {
-    font-size: $font-size-300;
+
+  @media (min-width: $screen-md) {
+    font-size: $font-size-500;
   }
 }
-h4 {
+h4, .heading-extra-small {
```

Comment:
when namespacing modifiers you should use an underscore for multi word names, ie .heading-extra_small